pkg/network/ovn_kubernetes.go

```diff
@@ -137,6 +140,31 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 		}
 	}
 
+	exportNetworkFlows := conf.ExportNetworkFlows
+	if exportNetworkFlows != nil {
+		if exportNetworkFlows.NetFlow != nil {
+			var collectors strings.Builder
+			for _, v := range exportNetworkFlows.NetFlow.Collectors {
+				collectors.WriteString(string(v) + ",")
```

Comment:
sometimes seen `Fprintf` used with `Builders`

```go
				fmt.Fprintf(&collectors, "%s,", v)`
```

etcd seems to use a style like:
```go
	var buf strings.Builder
	buf.WriteByte('(')
	for i := range sl {
		if i > 0 {
			buf.WriteByte(' ')
		}
		fmt.Fprint(&buf, sl[i])
	}
	buf.WriteByte(')')
```